### PR TITLE
refactor: fixes linting checks

### PR DESCRIPTION
**Pull request by thiagotnunes**
By running `./gradlew :sdks:java:io:google-cloud-platform:spotlessApply`

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/build.gradle [hunk 1]
```diff
@@ -201,3 +204,9 @@ task postCommit {
   dependsOn integrationTest
   dependsOn integrationTestKms
 }
+
+spotless {
+  java {
+    toggleOffOn()
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/build.gradle:210]: Enables off / on formatter in order to skip formatting of certain code snippets.

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/cdc/ReadChangeStreamPartitionDoFn.java [hunk 2]
```diff
@@ -157,10 +162,12 @@ public ProcessContinuation processElement(
         return done(partition, tracker);
       default:
         // TODO: Verify what to do here
-        throw new IllegalArgumentException("Unknown mode " + tracker.currentRestriction().getMode());
+        throw new IllegalArgumentException(
+            "Unknown mode " + tracker.currentRestriction().getMode());
     }
   }
 
+  // spotless:off
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/cdc/ReadChangeStreamPartitionDoFn.java:170]: We disabled the formatting, not to break the current ascii representation of the state machine (for the partition modes).

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/cdc/ReadChangeStreamPartitionDoFn.java [hunk 3]
```diff
@@ -195,50 +202,40 @@ public ProcessContinuation processElement(
    *                                      +------+
    *
    */
+  // spotless:on
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/cdc/ReadChangeStreamPartitionDoFn.java:205]: We re-enabled formatting from here on.

**Review by hengfengli — APPROVED** [review]
LGTM.

**Pull request merged**